pom.xml

```diff
@@ -349,8 +349,8 @@
           <version>1.1.0</version>
           <configuration>
             <cdapArtifacts>
-              <parent>system:cdap-data-pipeline[5.0.0,6.0.0-SNAPSHOT)</parent>
-              <parent>system:cdap-data-streams[5.0.0,6.0.0-SNAPSHOT)</parent>
+              <parent>system:cdap-data-pipeline[5.0.0,6.1.1]</parent>
```

Comment:
Please have the upper bound set to `7.0.0)`

Reply:
This is done